flowdb/bin/build/pg_config_template.conf

```diff
@@ -20,9 +20,9 @@
 #
 # Logging options
 #
-log_destination = 'csvlog'
+log_destination = '{log_destination}'
 log_directory = 'pg_log'
-logging_collector = 'on'
+logging_collector = '{collecter_on}'
 log_filename = 'postgres-%Y-%m-%d_%H%M%S'
 log_rotation_age = 1d
 log_rotation_size = 1GB
```

Comment:
Do we want to make log age + rotation size configurable via env var, or is it not worth it?

Reply:
I think not worth it via env var, because of the addition of #6601